packages/babel-plugin-transform-merge-sibling-variables/__tests__/fixtures/lift-different-declars/actual.js

```diff
@@ -1,3 +1,3 @@
 for (let i = 0; i < 0; i++) {
-  var i = 0;
```

Comment:
It is a syntax error since `i` has been declared. I have revised to declare `i` in block statement.